projects/igniteui-angular/src/lib/grids/selection/selection.service.ts

```diff
@@ -210,12 +210,12 @@ export class IgxGridCRUDService {
         }
 
         if (this.cellInEditMode) {
+            // TODO: case solely for f2/enter nav that uses enterEditMode as toggle. Refactor.
```

Comment:
Left this here, because this branch is only used for f2/enter handling as exit, which is not part of this methods job, so it's likely this belongs in the navigation service (along with the `isInCompositionMode` I assume). There's also a slight chance this could be part of `endEdit` too. I'll leave it for #8330